src/resources/views/crud/fields/checkbox.blade.php

```diff
@@ -1,30 +1,34 @@
 {{-- checkbox field --}}
 
 @php
-  $field['value'] = old_empty_or_null($field['name'], '') ??  $field['value'] ?? $field['default'] ?? '';
+  $field['value'] = old_empty_or_null($field['name'], '') ?? $field['value'] ?? $field['default'] ?? '';
+  $field['attributes']['class'] = ($field['attributes']['class'] ?? '') . ' form-check-input';
```

Comment:
Let's let the dev completely customize the class attribute, if they want:

```suggestion
  $field['attributes']['class'] = $field['attributes']['class'] ?? 'form-check-input';
```